src/encoded/commands/create_mapping_on_deploy.py

```diff
@@ -151,33 +155,50 @@ def get_deployment_config(app):
     current_prod_env = whodaman()  # current_prod_end = <current-data-env>
     my_env = get_my_env(app)
     deploy_cfg['ENV_NAME'] = my_env
-    if current_prod_env == my_env:
-        log.info('This looks like our production environment -- do not wipe ES')
-        deploy_cfg['WIPE_ES'] = False
-    elif my_env == guess_mirror_env(current_prod_env):
-        log.info('This looks like our staging environment -- do not wipe ES')
-        deploy_cfg['WIPE_ES'] = False  # do not wipe ES
+    deploy_cfg['SKIP'] = False  # set to True to skip the create_mapping step
+    deploy_cfg['STRICT'] = False
+    deploy_cfg['WIPE_ES'] = False
+    if my_env == guess_mirror_env(current_prod_env):
+        log.info('This looks like our staging environment -- wipe ES')
+        deploy_cfg['WIPE_ES'] = True
```

Comment:
This could also use the `apply_dict_overrides` from below, and the set of overrides can be a dictionary with a name like `PRODUCTION_OVERRIDES`, although now that I'm thinking of it, this could also use subclassing, so instead of initializing to `False` and then overriding, one could do:

```
DefaultDeploymentOptions:
    SKIP = False
    STRICT = False
    WIPE_ES = False

ProductionDeploymentOptions(DefaultDeploymentOptions):
    WIPE_ES = True
    STRICT = True

class HotseatDeploymentOptions(DefaultDeploymentOptions):
    SKIP = True
    STRICT = True

class OtherTestDeploymentOptions(DefaultDeploymentOptions):
    WIPE_ES = True
```
and then you do
```
options = GenericDeploymentOptions
if my_env == guess_mirror_env(current_prod_env):  # This function name is deprecated, so we'll want to fix that
    options = ProductionDeploymentOptions
...
```
But after playing it through, i realized it isn't really leveraging the class system enough, so I think just
```
DEFAULT_DEPLOYMENT_OPTIONS = {'SKIP': False, 'STRICT': False, 'WIPE_ES': False}
PRODUCTION_DEPLOYMENT_OPTION_OVERRIDES = {'WIPE_ES': True, 'STRICT': True}
HOTSEAT_DEPLOYMENT_OPTION_OVERRIDES = {'SKIP': True, 'STRICT': True}
OTHER_TEST_DEPLOYMENT_OPTION_OVERRIDES = {'WIPE_ES': True}
```
so that you can do:
```
deploy_cfg.= DEFAULT_DEPLOYMENT_OPTIONS.copy()
...
if my_env == guess_mirror_env(current_prod_env):  # This function name is deprecated, so we'll want to fix that
    deploy_cfg = apply_dict_overrides(deploy_cfg, **PRODUCTION_DEPLOYMENT_OPTION_OVERRIDES)
...
```